samcli/commands/pipeline/init/interactive_init_flow.py

```diff
@@ -18,6 +19,7 @@
 from samcli.lib.cookiecutter.template import Template
 from samcli.lib.utils import osutils
 from samcli.lib.utils.git_repo import GitRepo, CloneRepoException
+from samcli.lib.utils.osutils import copytree
```

Comment:
we have already imported `osutils` above and used it like `osutils.mkdir_temp`, can we unify the behavior? I vote for using `osutils.copytree`.